packages/now-cli/src/util/index.js

```diff
@@ -179,6 +179,7 @@ export default class Now extends EventEmitter {
         limits: nowConfig.limits,
         atlas,
         config: nowConfig,
+        functions: nowConfig.functions,
```

Comment:
Why is this a new key? The line above passes the whole `config` object 🤔

Reply:
I'm not sure either why the whole config gets passed here, probably for legacy reasons.

In Line 179 `limits` does the exact same thing.